src/clusterfuzz/_internal/bot/tasks/commands.py

```diff
@@ -304,10 +304,9 @@ def process_command(task):
         current_platform_id = environment.get_platform_id()
         testcase_platform_id = testcase.platform_id
 
-        # This indicates we are trying to run this job on the wrong platform.
-        # This can happen when you have different type of devices (e.g
-        # android) on the same platform group. In this case, we just recreate
-        # the task.
+        # This indicates we are trying to run this job on the wrong platform
+        # and potentially blocks fuzzing. See the 'subqueues' feature for
+        # more details: https://github.com/google/clusterfuzz/issues/3347
```

Comment:
Can we completely delete this code after you start using this?

Reply:
Ack on comment being useful to other teams, but I think that could go somewhere else potentially. 

I was more referring to the existing platform_id mechanism, which we can likely just get rid of after this right? 

Either way, this is just something for the future, so nothing actionable right now.

Reply:
Hypothetically yes, there will be no need to compare `current_platform_id` to  `testcase_platform_id` after this change as they will always be the same.